cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/OutputsUtil.java

```diff
@@ -33,4 +52,31 @@ private OutputsUtil() {
 
         return signatureReturnResultMap;
     }
+
+    public static void putResponseIn(Map<String, String> queryMapResult) {
+        if (queryMapResult != null) {
```

Comment:
@Tirla-Alin refactor that one in order to concatenate those 2 if's. Even though is hard to imagine a query that can produce a null response, the way that the code is written now doesn't handle that corner case.